zerver/openapi/zulip.yaml

```diff
@@ -16974,6 +16993,14 @@ paths:
             type: boolean
             default: true
           example: false
+        - name: exclude_archived
+          in: query
+          description: |
+            Excludes archived streams in realm.
```

Comment:
"Whether to exclude archived streams from the results."

The writing of the existing options isn't great on this page, but the "in realm" addition in particular is unnecessary/confusing, since no Zulip API lets you access data outside your realm.

Reply:
Fixed.